src/App.vue

```diff
@@ -48,35 +53,57 @@
 
 <script>
 import { getTubeLines, getStations, getPlatforms } from "./requests.js";
-
+import SingleDropdown from './components/singleDropdown.vue'
+import { Dedupe } from './utilities/dedupe.js'
```

Comment:
Can you make the casing consistent? getTubeLines is camel case but Dedupe is sentence case.